### PR TITLE
Refactoring model compression doc

### PR DESCRIPTION
**Pull request by chicm-ms**

### CODE REVIEW

File: docs/en_US/Compression/quantization.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#################
+
+Quantization refers to compressing models by reducing the number of bits required to represent weights or activations,
+which can reduce the computations and the inference time. In the context of deep neural networks, the major numerical
+format for model weights is 32-bit float, or FP32. Many research works have demonstrated that weights and activations
+can be represented using 8-bit integers without significant loss in accuracy. Even lower bit-widths, such as 4/2/1 bits,
+is an active field of research.
+
+A quntizer is a quantization algorithm implementation in NNI, NNI provides multiple quntizers as below. You can also
+create you own quntizer using NNI model compression inteface.
```
Comment (QuanluZhang) [inline docs/en_US/Compression/quantization.rst:12]: you -> your

File: docs/en_US/Compression/pruning.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#################
+Pruning
+#################
+
+A common technique to increase sparsity in neural network model weights and activations is pruning.
+The pruning methods explore the redundancy in the model weights(parameters) and try to remove/prune the redundant and uncritical weights.
+The redundant elements are pruned from the model, their values are zeroed and we make sure they don't take part in the back-propagation process.
+
+From pruning granularity perspective, fine-grained pruning refers to pruning each individual weights seperately.
```
Comment (colorjam) [inline docs/en_US/Compression/pruning.rst:9]: fine-grained pruning or unstructured pruning

File: docs/en_US/Compression/pruning.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#################
+Pruning
+#################
+
+A common technique to increase sparsity in neural network model weights and activations is pruning.
+The pruning methods explore the redundancy in the model weights(parameters) and try to remove/prune the redundant and uncritical weights.
+The redundant elements are pruned from the model, their values are zeroed and we make sure they don't take part in the back-propagation process.
+
+From pruning granularity perspective, fine-grained pruning refers to pruning each individual weights seperately. 
+Coase-grained pruning or structured pruning is pruning entire group of elements, such as convolutional filter.
```
Comment (colorjam) [inline docs/en_US/Compression/pruning.rst:10]: Coarse-grained

File: docs/en_US/Compression/pruning.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
+#################
+Pruning
+#################
+
+A common technique to increase sparsity in neural network model weights and activations is pruning.
+The pruning methods explore the redundancy in the model weights(parameters) and try to remove/prune the redundant and uncritical weights.
+The redundant elements are pruned from the model, their values are zeroed and we make sure they don't take part in the back-propagation process.
```
Comment (colorjam) [inline docs/en_US/Compression/pruning.rst:7]: It's better to use the same terminology (elements -> weights)

File: docs/en_US/Compression/pruning.rst [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+From pruning granularity perspective, fine-grained pruning refers to pruning each individual weights seperately. 
+Coase-grained pruning or structured pruning is pruning entire group of elements, such as convolutional filter.
+
+NNI provides multiple fine-grained pruning and structured pruning algorithms.
```
Comment (colorjam) [inline docs/en_US/Compression/pruning.rst:12]: suggest aligning the terminology, such as fine-grained and coarse-grained pruning vs. unstructured pruning and structured pruning.

File: docs/en_US/model_compression.rst [hunk 6]
```diff
@@ -2,7 +2,15 @@
 Model Compression
 #################
 
-NNI provides an easy-to-use toolkit to help user design and use compression algorithms.
+Deep neural networks (DNNs) have achieved great success in many tasks. However, typical neural networks are both
+computationally expensive and energy intensive, can be difficult to be deployed on devices with low computation
+resources or with strict latency requirements. Therefore, a natural thought is to perform model compression to
+reduce model size and accelarete model training/inference without losing performance significantly. Model compression
```
Comment (colorjam) [inline docs/en_US/model_compression.rst:8]: accelarete -> accelerate

File: docs/en_US/Compression/quantization.rst (hunk 1 above)
Comment (chicm-ms) [reply docs/en_US/Compression/quantization.rst]: Thanks, updated.

File: docs/en_US/Compression/pruning.rst (hunk 2 above)
Comment (chicm-ms) [reply docs/en_US/Compression/pruning.rst]: Thanks, updated.

File: docs/en_US/Compression/pruning.rst (hunk 3 above)
Comment (chicm-ms) [reply docs/en_US/Compression/pruning.rst]: Thanks, fixed.

File: docs/en_US/Compression/pruning.rst (hunk 4 above)
Comment (chicm-ms) [reply docs/en_US/Compression/pruning.rst:7]: Thanks, fixed.

File: docs/en_US/model_compression.rst (hunk 6 above)
Comment (chicm-ms) [reply docs/en_US/model_compression.rst]: Thanks, fixed.

File: docs/en_US/Compression/pruning.rst (hunk 5 above)
Comment (chicm-ms) [reply docs/en_US/Compression/pruning.rst]: Thanks, fixed.

**Review by QuanluZhang — APPROVED** [review]

File: docs/en_US/Compression/pruning.rst [hunk 7]
```diff
@@ -0,0 +1,26 @@
+#################
+Pruning
+#################
+
+A common technique to increase sparsity in neural network model weights and activations is pruning.
```
Comment (zheng-ningxin) [inline docs/en_US/Compression/pruning.rst:5]: Suggests changing to a simple sentence，`increase sparsity` sounds a little bit strange? how about:
Pruning is a common technique for compressing models.

Comment (chicm-ms) [reply docs/en_US/Compression/pruning.rst]: Thanks for good suggestion, updated.

File: docs/en_US/Compression/pruning.rst [hunk 8]
```diff
@@ -0,0 +1,26 @@
+#################
+Pruning
+#################
+
+Pruning is a common technique to compress neural network models.
+The pruning methods explore the redundancy in the model weights(parameters) and try to remove/prune the redundant and uncritical weights.
+The redundant elements are pruned from the model, their values are zeroed and we make sure they don't take part in the back-propagation process.
+
+From pruning granularity perspective, fine-grained pruning or unstructured pruning refers to pruning each individual weights seperately.
```
Comment (zheng-ningxin) [inline docs/en_US/Compression/pruning.rst:9]: seperately -> separately

File: docs/en_US/Compression/pruning.rst [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+Coarse-grained pruning or structured pruning is pruning entire group of weights, such as convolutional filter.
```
Comment (zheng-ningxin) [inline docs/en_US/Compression/pruning.rst:10]: such as a convolutional filter.

File: docs/en_US/Compression/quantization.rst [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+Quantization
+#################
+
+Quantization refers to compressing models by reducing the number of bits required to represent weights or activations,
+which can reduce the computations and the inference time. In the context of deep neural networks, the major numerical
+format for model weights is 32-bit float, or FP32. Many research works have demonstrated that weights and activations
+can be represented using 8-bit integers without significant loss in accuracy. Even lower bit-widths, such as 4/2/1 bits,
+is an active field of research.
+
+A quntizer is a quantization algorithm implementation in NNI, NNI provides multiple quntizers as below. You can also
```
Comment (zheng-ningxin) [inline docs/en_US/Compression/quantization.rst:11]: quntizer -> quantizer

File: docs/en_US/Compression/quantization.rst [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+create your own quntizer using NNI model compression interface.
+
+..  toctree::
+    :maxdepth: 2
+
+    Quntizers <Quantizer>
```
Comment (zheng-ningxin) [inline docs/en_US/Compression/quantization.rst:17]: quntizer -> quantizer

File: docs/en_US/Overview.md [hunk 12]
```diff
@@ -63,9 +63,11 @@ NNI has support for many one-shot NAS algorithms such as ENAS and DARTS through
 Other than one-shot NAS, NAS can also run in a classic mode where each candidate architecture runs as an independent trial job. In this mode, similar to hyperparameter tuning, users have to start an NNI experiment and choose a tuner for NAS.
 
 ### Model Compression
-Model Compression on NNI includes pruning algorithms and quantization algorithms. These algorithms are provided through NNI trial SDK. Users can directly use them in their trial code and run the trial code without starting an NNI experiment. A detailed description of model compression and its usage can be found [here](Compressor/Overview.md).
+NNI provides an easy-to-use model compression framework to compress deep neural networks, the compressed networks typically have much smaller model size and much faster
+inference speed without losing performance significally. Model compression on NNI includes pruning algorithms and quantization algorithms. NNI provides many pruning and
```
Comment (zheng-ningxin) [inline docs/en_US/Overview.md:67]: significally -> significantly

File: docs/en_US/model_compression.rst [hunk 13]
```diff
@@ -2,7 +2,15 @@
 Model Compression
 #################
 
-NNI provides an easy-to-use toolkit to help user design and use compression algorithms.
+Deep neural networks (DNNs) have achieved great success in many tasks. However, typical neural networks are both
+computationally expensive and energy intensive, can be difficult to be deployed on devices with low computation
+resources or with strict latency requirements. Therefore, a natural thought is to perform model compression to
+reduce model size and accelerete model training/inference without losing performance significantly. Model compression
```
Comment (zheng-ningxin) [inline docs/en_US/model_compression.rst:8]: accelerete -> accelerate

**Review by zheng-ningxin — APPROVED** [review]

File: docs/en_US/Compression/pruning.rst (hunk 8 above)
Comment (chicm-ms) [reply docs/en_US/Compression/pruning.rst]: Thanks, fixed.

File: docs/en_US/Compression/pruning.rst (hunk 9 above)
Comment (chicm-ms) [reply docs/en_US/Compression/pruning.rst]: Thanks, fixed.

File: docs/en_US/Compression/quantization.rst (hunk 11 above)
Comment (chicm-ms) [reply docs/en_US/Compression/quantization.rst]: Thanks, fixed.

File: docs/en_US/Overview.md (hunk 12 above)
Comment (chicm-ms) [reply docs/en_US/Overview.md]: Thanks, fixed.

File: docs/en_US/model_compression.rst (hunk 13 above)
Comment (chicm-ms) [reply docs/en_US/model_compression.rst]: Thanks, fixed.

**Review by colorjam — APPROVED** [review]

**Review by SparkSnail — APPROVED** [review]

**Pull request merged**